.openpublishing.redirection.azure.json

```
@@ -102,7 +102,7 @@
         },
         {
             "source_path_from_root": "/docs/azure/sdk/authentication-azure-hosted-apps.md",
-            "redirect_url": "/dotnet/azure/sdk/authentication/azure-hosted-apps"
+            "redirect_url": "/docs/azure/sdk/authentication/system-assigned-managed-identity"
```

@gewarren Is that incorrect?  Should it be:

```suggestion
            "redirect_url": "/dotnet/azure/sdk/authentication/system-assigned-managed-identity"
```

---

@BillWagner Yes, it's because line 133 is incorrect.